added the experimental note for the plaintext-only value of contenteditable global attribute (#36750)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

Reviewer: Oh and we'll also need an entry here: https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/133#experimental_web_features. (thank you!)

## HTML

> ### contenteditable="plaintext-only"

Reviewer: This section can be moved to after [Toggle password display](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#toggle_password_display) because in every tech area, we're following the order of older to newer releases

Author: done

> ### contenteditable="plaintext-only"

Reviewer: how about 
```suggestion
### Plaintext-only contenteditable mode
```

> The [`contenteditable`](/en-US/docs/Web/HTML/Global_attributes/contenteditable) value `plaintext-only` indicates that an element is editable, but rich text formatting is disabled and the formatting of pasted text is stripped. (See [Firefox bug 1922723](https://bugzil.la/1922723) for more details.)

Reviewer: To place emphasis on the value that is now supported, we could move it upfront. Plus some other nits. What do you think about:
```suggestion
The `plaintext-only` value of the [`contenteditable`](/en-US/docs/Web/HTML/Global_attributes/contenteditable) global attribute indicates that the element is editable; rich text formatting is disabled and any formatting in pasted text is automatically stripped. (See [Firefox bug 1922723](https://bugzil.la/1922723) for more details.)
```

<table>
  <thead>
    <tr>
      <th>Release channel</th>
      <th>Version added</th>
      <th>Enabled by default?</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>Nightly</th>

>       <td>134</td>

Reviewer: this would be 133

Author: done
